### PR TITLE
Add Google font

### PR DESCRIPTION
**Pull request by Yonela986**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,6 +24,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (Loreen-netizen) [inline public/index.html:27]: This is an example of a request review

**Review by Loreen-netizen — CHANGES_REQUESTED** [review]
see comment about google font

**Comment by schalkventer** [comment]
@Loreen-netizen has a good point, please change

**Comment by schalkventer** [comment]
This is a test comment

**Review by schalkventer — APPROVED** [review]
Happy with changes

**Pull request merged**